src/styles/blocks/card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  box-sizing: border-box;
+  height: $card-height;
+  width: $card-width;
+  border: 1px solid $border-color;
```

Comment:
The variable `$border-color` is used for the `border` property. Ensure that this variable is defined elsewhere in your SCSS files. If it's not defined, you need to declare it with a valid color value before using it.